app/containers/Questions/Content/Body/index.js

```diff
@@ -116,6 +117,7 @@ Body.propTypes = {
   displayTopQuestionMove: PropTypes.bool,
   topQuestionActionProcessing: PropTypes.bool,
   isPromoted: PropTypes.bool,
+  isCommunityFeed: PropTypes.bool,
 };
 
 export default memo(Body);
```

Comment:
remove function memo, this is a simple component, nothing needs to be optimized in it

Reply:
+